spock-core/src/main/java/org/spockframework/runtime/extension/builtin/TimeoutInterceptor.java

```diff
@@ -123,4 +130,94 @@ public void run() {
       throw saved;
     }
   }
+
+  private void logUnsuccessfulInterrupt(String methodName, long now, long timeoutAt, long waitMillis, int unsuccessfulAttempts) {
+    System.err.printf(
+      "[spock.lang.Timeout] Method '%s' has not stopped after timing out %1.2f seconds ago - interrupting. Next try in %1.2f seconds.\n%n",
+      methodName,
+      Duration.ofNanos(now - timeoutAt).toMillis() / 1000.,
+      waitMillis / 1000.
```

Comment:
🧐 Use `d` literal instead of `.`
```suggestion
      Duration.ofNanos(now - timeoutAt).toMillis() / 1000d,
      waitMillis / 1000d
```

Reply:
Done in https://github.com/spockframework/spock/pull/1658/commits/b71e5c852fcc390ed3e2ca32f5dd5de173b39150